apps/examples/bluetooth/bt_unit_test.c

```diff
@@ -265,6 +256,32 @@ static void __le_scan_result_cb(int result, bt_adapter_le_device_scan_result_inf
 	}
 }
 
+static void __state_changed_cb(int result, bt_adapter_state_e adapter_state, void *user_data)
+{
+	int ret;
+	char *name = NULL;
+
+	TC_PRT("__state_changed_cb\n");
+	TC_PRT("result: %s\n", __bt_get_error_message(result));
+	TC_PRT("state: %s\n", (adapter_state == BT_ADAPTER_ENABLED) ? "ENABLED" : "DISABLED");
+
+//	adapter_enabled = ((adapter_state == BT_ADAPTER_ENABLED) ? true : false);
+
+	ret = bt_adapter_get_name(&name);
```

Comment:
Can we check adapter is enabled or not before calling remaining functions?

Reply:
Done. Thank you.